docker/Dockerfile

```diff
@@ -1,15 +1,24 @@
 FROM debian:jessie
 
-ENV POSTGREST_VERSION 0.4.1.0
-
-RUN apt-get update && \
-    apt-get install -y tar xz-utils wget libpq-dev && \
-    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
-
-RUN wget http://github.com/begriffs/postgrest/releases/download/v${POSTGREST_VERSION}/postgrest-${POSTGREST_VERSION}-ubuntu.tar.xz && \
-    tar --xz -xvf postgrest-${POSTGREST_VERSION}-ubuntu.tar.xz && \
+# Install libpq5
+RUN apt-get -qq update && \
+    apt-get -qq install -y --no-install-recommends libpq5 && \
+    apt-get -qq clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Install postgrest
+RUN POSTGREST_VERSION="0.4.1.0" \
+    BUILD_DEPS="curl ca-certificates xz-utils" && \
+    apt-get -qq update && \
```

Comment:
What's the reason for this second apt-get update? Just because this RUN statement is a separate stage from the previous one and may be run later on independently?